zospy/functions/nce.py

```diff
@@ -33,3 +32,73 @@ def object_change_type(obj: _ZOSAPI.Editors.NCE.INCERow, new_type: constants.Edi
     # Apply
     new_surface_type_settings = obj.GetObjectTypeSettings(new_type)
     obj.ChangeType(new_surface_type_settings)
+
+def find_object_comment(nce: _ZOSAPI.Editors.NCE, comment: str, case_sensitive: bool=False) -> list[int] | int:
+    """Function equivalent to ZPL numeric function OBJC($A).
```

Comment:
Please do not refer to ZPL macros here, give a description of the function behaviour instead